pkg/profiles/openapi.go

```diff
@@ -104,11 +108,18 @@ func swaggerToServiceProfile(swagger spec.Swagger, namespace, name, clusterDomai
 	return profile
 }
 
-func mkRouteSpec(path, pathRegex string, method string, responses *spec.Responses) *sp.RouteSpec {
+func mkRouteSpec(path, pathRegex string, method string, operation *spec.Operation) *sp.RouteSpec {
+	retryable := false
+	var responses = (*spec.Responses)(nil)
+	if operation != nil {
+		retryable, _ = operation.VendorExtensible.Extensions.GetBool(xLinkerdRetryable)
```

Comment:
@kohsheen1234 still wondering about the return value from `item.Delete.VendorExtensible.Extensions.GetBool`

Is there anything that we should do if the second value is `false`?

Reply:
@cpretzer The second return value is not of concern to us, we need not do anything if the send value is `false`.

Reply:
LGTM, thanks for confirming